app/pages/Multisig/index.js

```diff
@@ -0,0 +1,183 @@
+import React, {Component} from "react";
+import { connect } from 'react-redux';
+import "./multisig.scss";
+import nodeClient from "../../utils/nodeClient";
+import {showError, showSuccess} from "../../ducks/notifications";
+import PropTypes from "prop-types";
+import {I18nContext} from "../../utils/i18n";
+import walletClient from "../../utils/walletClient";
+import HDPublicKey from "hsd/lib/hd/public";
+import { getPassphrase, signMessage } from "../../ducks/walletActions";
+import { MTX } from "hsd/lib/primitives";
+
+@connect(
+  (state) => ({
+  }),
+  (dispatch) => ({
+    showError: (message) => dispatch(showError(message)),
+    showSuccess: (message) => dispatch(showSuccess(message)),
+    signMessage: (message) => dispatch(signMessage(message)),
+  }),
+)
+class Multisig extends Component {
+  static propTypes = {
+    showError: PropTypes.func.isRequired,
+    showSuccess: PropTypes.func.isRequired,
+  };
+
+  static contextType = I18nContext;
+
+  state = {
+    xpubKey: '',
+    accountInfo: null,
+    addKey: '',
+    rawMessage: '',
+    signature: '',
+    verified: null,
+  };
+
+  async componentDidMount() {
+    await this.reloadAccount();
+  }
+
+  broadcast = async () => {
+    var tx = null;
+
+    try {
+      tx = MTX.decode(Buffer.from(this.state.signature, 'hex'));
+    }
+    catch(e) {
+      alert('Invalid TX');
+      return;
+    }
+
+    try {
+      const txObj = tx.toJSON();
+      await nodeClient.broadcastRawTx(txObj.hex);
+      alert('Broadcast success. If the transaction is valid it will be sent soon. hash=' + txObj.hash);
+    }
+    catch(e) {
+      alert('Failed to broadcast: ' + e?.message);
+    }
+  };
+
+  addSignature = async () => {
+    const {signMessage} = this.props;
+    var tx = null;
+
+    try {
+      tx = MTX.decode(Buffer.from(this.state.rawMessage, 'hex'));
+    }
+    catch(e) {
+      alert('Invalid TX');
+      return;
+    }
+    const resp = await signMessage(tx);
+    this.setState({signature: resp.hex});
+  }
+
+  reloadAccount = async () => {
+      const accountInfo = await walletClient.getAccountInfo();
+      this.setState({accountInfo: accountInfo, xpubKey: accountInfo.accountKey});
+  };
```

Comment:
I don't think we'll need this if we put entire `accountInfo` object in the state (https://github.com/kyokan/bob-wallet/pull/493#discussion_r841927950)